Overview.src.html

```diff
@@ -6243,7 +6243,10 @@ <h4 id="normalizing-a-sequence-of-keyframes">Normalizing a sequence of keyframes
 section></a> is performed before this operation is
 initiated.
 
-1.  If <var>frames</var> is a single {{Keyframe}} object, replaces
+1.  If <var>frames</var> is a {{SharedKeyframeList}} object, then
+    replace <var>frames</var> with the {{Keyframe}} or
+    sequence used to construct the {{SharedKeyframeList}}.
```

Comment:
The SharedKeyframeList is currently readonly and immutable, so there's no observable difference. There isn't currently a way to get editable keyframes out of a KeyframeEffect, so modifications can't occur here; I would expect setFrames to locally override rather than propagating back to the list.

If we change the immutability of SharedKeyframeList later we have a number of options, including observing this change and triggering setFrames on impacted KeyframeEffects. It's not clear that we'll necessarily ever want SharedKeyframeList objects to be immutable though.